corehq/apps/reports/filters/users.py

```diff
@@ -176,7 +176,7 @@ class ExpandedMobileWorkerFilter(BaseMultipleOptionFilter):
     placeholder = ugettext_lazy(
         "Specify groups and users to include in the report")
     is_cacheable = False
-    options_url = 'emwf_options'
```

Comment:
I think this means `Locations` will start showing up in the dropdown for all reports, right?